src/components/AllGames/GamesContainer/GamesContainer.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './GamesContainer.module.css'
+
+import GameCard from '../GameCard/GameCard'
+import { MOBILE_WIDTH, TABLET_WIDTH } from '../../../shared/constants'
+
+GamesContainer.propTypes = {
+    icon: PropTypes.string,
+    numberToDisplay: PropTypes.number,
+    title: PropTypes.string.isRequired,
+    games: PropTypes.array.isRequired,
+}
+
+function GamesContainer({ title, games, numberToDisplay = null, icon = null }) {
+    const [width, setWindowWidth] = useState(0)
+
+    useEffect(() => {
+        updateScreenWidth()
+
+        window.addEventListener('resize', updateScreenWidth)
+        return () => window.removeEventListener('resize', updateScreenWidth)
+    }, [])
+
+    const updateScreenWidth = () => setWindowWidth(window.innerWidth)
+
+    const gamesToDisplay = () => {
+        if (!numberToDisplay) return games
+
+        let length = 4
+        if (width <= MOBILE_WIDTH) length = 2
```

Comment:
2 per row, it doesn't mean that we have to show **only** 2 games

Reply:
It is implemented, you just do not need to pass numberToDisplay prop into the game container